bin/load-runner.js

```diff
@@ -182,21 +193,35 @@ var l = new loop.MultiLoop({
 
     var scriptArgs = [args.s].concat(args._);
 
-    scriptArgs.forEach(function(arg, i) {
-      if (arg === '{runNum}') {
-        scriptArgs[i] = curRuns;
-      }
-    });
+    scriptArgs.forEach(
+      function(arg, i) {
+        switch (arg) {
+          case '{runNum}': {
+            scriptArgs[i] = curRuns;
+            break;
+          }
+        }
+      });
+
+    /**
+     * Environment variables for the test script.
+     */
+    var env = {
+      'LR_RUN_NUMBER': curRuns,
+      'LR_RAND': rand(),
+      'LR_RUN_COUNT': args.n
```

Comment:
Minor suggestion: change `args.n` here to be `args.numUsers` -- it's a little more descriptive, and should just work, since the alias is set.

Also, do you think `LR_RUN_COUNT` might be better as `LR_TOTAL_RUNS`?